integrations/standalone/tests/integration/parts/signal-catch.ts

```diff
@@ -1,38 +1,46 @@
-import { expect } from '@playwright/test';
 import { Part } from '../../pageobjects/Part';
-import { ComboboxUtil } from '../utils/combobox-util';
-import { PartTest } from './part-tester';
+import { NewPartTest, PartObject } from './part-tester';
+import { Combobox } from '../../pageobjects/Combobox';
+import { Checkbox } from '../../pageobjects/Checkbox';
 
-export class SignalCatchTester implements PartTest {
-  constructor(private readonly makroSupport: boolean = false) {}
+class SignalCatch extends PartObject {
+  signal: Combobox;
+  attach: Checkbox;
 
-  partName() {
-    return 'Signal';
+  constructor(part: Part, private readonly makroSupport: boolean = false) {
```

Comment:
markoSupport -> attachToBusinessCaseSupport?

Reply:
This technically does not only add the checkbox, but also adds a macroinput to the singal, but as this is currently a combobox, I'm not sure how to implement that in the UI...